docs/pt/docs/advanced/security/http-basic-auth.md

```diff
@@ -68,35 +40,7 @@ Para lidar com isso, primeiramente nós convertemos o `username` e o `password`
 
 Então nós podemos utilizar o `secrets.compare_digest()` para garantir que o `credentials.username` é `"stanleyjobson"`, e que o `credentials.password` é `"swordfish"`.
 
-//// tab | Python 3.9+
-
-```Python hl_lines="1  12-24"
-{!> ../../docs_src/security/tutorial007_an_py39.py!}
-```
-
-////
-
-//// tab | Python 3.8+
-
-```Python hl_lines="1  12-24"
-{!> ../../docs_src/security/tutorial007_an.py!}
-```
-
-////
-
-//// tab | Python 3.8+ non-Annotated
-
-/// tip | "Dica"
-
-Prefira utilizar a versão `Annotated` se possível.
-
-///
-
-```Python hl_lines="1  11-21"
-{!> ../../docs_src/security/tutorial007.py!}
-```
-
-////
+{* ../../docs_src/security/tutorial007_an_py39.py hl[12:24] *}
```

Comment:
missed the first line hl:

```
{* ../../docs_src/security/tutorial007_an_py39.py hl[1,12:24] *}
```

Reply:
Thank you for the feedback! I've made the requested changes.